components/src/main/res/layout/component_menu_panel_row.xml

```diff
@@ -0,0 +1,92 @@
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <LinearLayout
+        android:id="@+id/layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentTop="true"
+        android:focusable="true"
+        android:orientation="vertical">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content">
+
+            <TextView
+                android:id="@+id/text_title"
+                style="@style/Text.Menu"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:importantForAccessibility="no"
+                android:gravity="start"
+                android:paddingStart="@dimen/hmrc_spacing_16"
+                android:paddingTop="@dimen/hmrc_spacing_24"
+                android:paddingEnd="@dimen/hmrc_spacing_8"
+                android:paddingBottom="@dimen/hmrc_spacing_24"
+                app:layout_constrainedWidth="true"
+                app:layout_constraintEnd_toStartOf="@id/notification"
+                app:layout_constraintHorizontal_bias="0"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
+                android:id="@+id/notification"
+                style="@style/Text.Bold"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:importantForAccessibility="no"
+                android:layout_marginEnd="@dimen/hmrc_spacing_16"
+                android:layout_marginTop="@dimen/hmrc_spacing_24"
+                android:gravity="center"
+                android:textColor="@color/hmrc_white"
+                android:visibility="gone"
+                app:layout_constraintDimensionRatio="1:1"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+        <TextView
+            android:id="@+id/text_body"
+            style="@style/Text.Body"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:importantForAccessibility="no"
+            android:gravity="start"
+            android:paddingStart="@dimen/hmrc_spacing_16"
+            android:paddingEnd="@dimen/hmrc_spacing_16"
+            android:paddingBottom="@dimen/hmrc_spacing_24" />
+
+    </LinearLayout>
+
+        <ImageView
```

Comment:
The indentation here isn't quite right. Could do with a quick "Auto-Indent lines" :)